src/elm/Pages/Admin/Workers.elm

```diff
@@ -0,0 +1,430 @@
+                    , query =
+                        Dict.update "page" (\_ -> Just <| String.fromInt pageNumber) route.query
+                    , hash = route.hash
+                    }
+                , Effect.getWorkers
+                    { baseUrl = shared.velaAPIBaseURL
+                    , session = shared.session
+                    , onResponse = GetWorkersResponse
+                    , pageNumber = Just pageNumber
+                    , perPage = Dict.get "perPage" route.query |> Maybe.andThen String.toInt
+                    }
+                ]
+            )
+
+        -- REFRESH
+        Tick options ->
+            ( model
+            , Effect.getWorkers
+                { baseUrl = shared.velaAPIBaseURL
+                , session = shared.session
+                , onResponse = GetWorkersResponse
+                , pageNumber = Dict.get "page" route.query |> Maybe.andThen String.toInt
+                , perPage = Dict.get "perPage" route.query |> Maybe.andThen String.toInt
+                }
+            )
+
+
+
+-- SUBSCRIPTIONS
+
+
+{-| subscriptions : takes model and returns subscriptions.
+-}
+subscriptions : Model -> Sub Msg
+subscriptions model =
+    Interval.tickEveryFiveSeconds Tick
+
+
+
+-- VIEW
+
+
+{-| view : takes models, route, and creates the html for the page.
+-}
+view : Shared.Model -> Route () -> Model -> View Msg
+view shared route model =
+    { title = ""
+    , body =
+        [ viewWorkers shared model route
+        , Components.Pager.view
+            { show = True
+            , links = model.pager
+            , labels = Components.Pager.defaultLabels
+            , msg = GotoPage
+            }
+        ]
+    }
+
+
+{-| viewWorkers : renders a list of workers.
+-}
+viewWorkers : Shared.Model -> Model -> Route () -> Html Msg
+viewWorkers shared model route =
+    let
+        actions =
+            Just <|
+                div [ class "buttons" ]
+                    [ Components.Pager.view
+                        { show = True
+                        , links = model.pager
+                        , labels = Components.Pager.defaultLabels
+                        , msg = GotoPage
+                        }
+                    ]
+
+        ( noRowsView, rows ) =
+            let
+                viewHttpError e =
+                    span [ Util.testAttribute "workers-error" ]
+                        [ text <|
+                            case e of
+                                Http.BadStatus statusCode ->
+                                    case statusCode of
+                                        401 ->
+                                            "No workers found"
+
+                                        _ ->
+                                            "No workers found, there was an error with the server (" ++ String.fromInt statusCode ++ ")"
+
+                                _ ->
+                                    "No workers found"
```

Comment:
curious about the `401` message. did you intentionally not call out auth?

Reply:
not sure, i think @ecrupper wrote that bit. i'll plop a message in there, good catch TY

Reply:
lol i didnt mean to call him out, i was just pointing it out in case it was intentional